packages/dashboard-frontend/src/Layout/Navigation/MainList.tsx

```diff
@@ -17,15 +17,22 @@ import { NavigationItemObject } from '.';
 import { ROUTE } from '../../route.enum';
 import { AppState } from '../../store';
 import { connect, ConnectedProps } from 'react-redux';
-import { selectAllWorkspacesNumber } from '../../store/Workspaces/selectors';
+import { selectAllWorkspaces } from '../../store/Workspaces/selectors';
 
 type Props = MappedProps & {
   activePath: string;
 };
 
 export class NavigationMainList extends React.PureComponent<Props> {
   private get items(): NavigationItemObject[] {
-    const { allWorkspacesNumber } = this.props;
+    const { allWorkspaces } = this.props;
+
+    const allWorkspacesNumber = allWorkspaces.filter(workspace => {
+      if (workspace.isDevWorkspace) {
+        return true;
+      }
+      return (workspace.ref as che.Workspace).attributes?.converted === undefined;
```

Comment:
I think that all converted che-based workspaces won`t be shown in che-server mode

Reply:
I've added some tests to be sure that workspaces are filtered correctly: [`56af2c1` (#441)](https://github.com/eclipse-che/che-dashboard/pull/441/commits/56af2c1cc519a5fa2565372ab1147b9d4e1ab910#diff-72a5682566ebc140e03edbf8fa0216b59787c596933997e1dc25a1f3badec8f3R90-R157)